WatchApp Extension/ComplicationController.swift

```diff
@@ -40,10 +40,13 @@ final class ComplicationController: NSObject, CLKComplicationDataSource {
     
     // MARK: - Timeline Population
 
-    private lazy var formatter = NumberFormatter()
+ //   private lazy var formatter = NumberFormatter()
 
     func getCurrentTimelineEntry(for complication: CLKComplication, withHandler handler: (@escaping (CLKComplicationTimelineEntry?) -> Void)) {
 
+        let formatter = NumberFormatter(); formatter.usesSignificantDigits = true; formatter.minimumSignificantDigits = 2; formatter.maximumSignificantDigits = 3
```

Comment:
What does this display with mg/dl?

Need to avoid repeating this code three times.